freeipa/src/main/resources/freeipa-salt/salt/monitoring/scripts/cipa.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
is there any python 2/3 specific code? if yes, shouldn't this point to the right python binary? or we are confident enough that this is always right even on older instances

Reply:
actually this runs properly with both python 2 and 3, but can be more specific of course